susie/src/main/resources/install-susie.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash -xe
+
+# susie method
+## Developed with python 3 and R
+
+SuSiE_ROOT=/mnt/var/susie
+
+# install to the root directory
+sudo mkdir -p "$SuSiE_ROOT"
+cd "$SuSiE_ROOT"
+
+# install yum dependencies
+sudo yum install -y python3-devel
+sudo yum install -y R
```

Comment:
Do you need to install R? I thought it was installed by default on the boxes.

Reply:
No longer needed to install R. The current version is good now.